doc/api/next_api_changes/2019-02-07-AH.rst

```diff
@@ -0,0 +1,6 @@
+Modify output of Axes.hist when input consists of multiple empty lists
+``````````````````````````````````````````````````````````````````````
+
+Input that consists of multiple empty lists will now return a set of histogram
+values for each one of the lists. For example, an input of ``[[],[]]`` will
+return 2 sets of histogram values. Previously, a single set was returned.
```

Comment:
"two lists" / "single list" (a "set" is something else...)

Reply:
Fixed, thanks.